src/Illuminate/Contracts/Encryption/Encrypter.php

```diff
@@ -21,4 +21,19 @@ public function encrypt($value, $serialize = true);
      * @return mixed
      */
     public function decrypt($payload, $unserialize = true);
+
+    /**
+     * Generate a new key for the chosen cipher.
+     *
+     * @param $cipher
+     * @return mixed
+     */
+    public function generateKey($cipher = null);
```

Comment:
your implementation appears to accept 2 parameters, a 'driver' and a 'cipher'. I believe one of these needs to be updated.

Reply:
@browner12 thank you for pointing this out, the `EncryptionManager` should now conform to the contract